packages/expo-file-system/ios/Next/FileSystemFile.swift

```diff
@@ -112,4 +126,25 @@ internal final class FileSystemFile: FileSystemPath {
     try validatePermission(.read)
     return try Data(contentsOf: url).base64EncodedString()
   }
+
+  func getInfoAsync(options: InfoOptions) throws -> FileInfo {
+    try validatePermission(.read)
+    switch url.scheme {
+    case "file":
+      let result = FileInfo()
+      if try exists {
+        result.exists = true
+        result.isDirectory = false
+        result.uri = url.absoluteString
+        result.size = try size
+        result.modificationTime = try modificationTime
+      }
+      if options.md5 {
+        result.md5 = try md5
+      }
+      return result
+    default:
+      throw UnableToGetAsyncInfoException("url scheme is not supported")
```

Comment:
We can add in the scheme that we detected. May be easier to track down issues for the users (and us if they report an issue).
```suggestion
      throw UnableToGetAsyncInfoException("url scheme `\(url.scheme)` is not supported")
```